apis/phabricator.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, config):
             self.url = config['url']
 
     @logEntryExit
-    @retry
+    @retry(times=RETRY_TIMES)
```

Comment:
Passing `RETRY_TIMES` here seems a bit redundant. Can this just be left out? Or would the default argument in the `retry` function not pick up the updated `RETRY_TIMES` then? In that case, I would just remove the `times` argument from the `retry` function, and use `RETRY_TIMES` directly, as we don't seem to be using those arguments anyway. You could also [override the default arguments](https://stackoverflow.com/questions/17625695/is-it-possible-to-change-a-functions-default-parameters-in-python) of retry in the tests, but that seems a bit sketchy to me.

But looks good to me besides that.

Reply:
Thanks, that does make it a bit cleaner.